Makefile

```diff
@@ -217,6 +217,10 @@ tools/bin/xprog: tools/check/xprog.go
 	cd tools/check; \
 	$(GO) build -o ../bin/xprog xprog.go
 
+tools/check/go.mod: tools/check/check.mod tools/check/check.sum
```

Comment:
What is the reason of renaming `go.mod` to `check.mod`?

Reply:
It is a workaround to prevent the SCA tool from scanning linter's third-party dependencies as project dependencies.